data/json/items/tool/cooking.json

```diff
@@ -143,6 +144,7 @@
     "symbol": ";",
     "color": "light_gray",
     "ammo": [ "charcoal" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "charcoal": 500 } } ],
```

Comment:
it says max charges 200 below, this is changing the behavior to something new

Reply:
Thanks I'll fix that.

Reply:
I figured out what the problem is in #41435 the furniture charcoal smoker calls to the portable charcoal smoker instead of creating a pseudo item.  This explains the discrepancy between descriptions. I'll create an f_char_smoker that holds enough charges to smoke the amount of food it can hold.